### DISCUSSION TITLE
[Feature Request] Add a drawer feature and  datetimepicker

### DISCUSSION THREAD

**Discussion by kiplagatcollins** [heroui-inc/heroui#1897] [Feature requests]
### Is your feature request related to a problem? Please describe.

Add Drawer like a modal for the side 

### Describe the solution you'd like

Add Drawer like a modal for the side 

### Describe alternatives you've considered

Add Drawer like a modal for the side 

### Screenshots or Videos

Add Drawer like a modal for the side

**Comment by e-Naeim** [comment]
datetime picker is really needed, I hope you are going to do it
